### ISSUE TITLE
Folder View for adding songs to playlist

### ISSUE THREAD

**Issue by posLop** [Iamlooker/Howl#14]
I have lots of songs that i sort by folder and when i import them i cant bulk add them to a playlist.  there needs to be some type of feature to either make playlists based off of folder structure or have a view where i can select a folder and recusrively add all the songs inside to a playlist

**Comment by Iamlooker** [comment]
Playlist implementation is under active development

**Comment by Pranab009** [comment]
![Screenshot_20220713-232156](https://user-images.githubusercontent.com/65340361/178798691-b0ed6fdc-8bf5-4c73-a463-678e649536a6.png)
See this image make this type album show folders. Or add a setting

**Comment by Pranab009** [comment]
See this image make this type album show folders. Or add a setting.

![Uploading Screenshot_20220713-232156.png…]()

![Screenshot_20220713-232532](https://user-images.githubusercontent.com/65340361/178799651-549fab54-e016-4d6a-bc11-472068d05280.png)

![Screenshot_20220713-232542](https://user-images.githubusercontent.com/65340361/178799670-943ffbff-3791-4cdf-90a1-496167cb00df.png)

**Comment by iBabu007** [comment]
See this image make this type album show folders. Or add a setting.

![Screenshot_20220713-232156](https://user-images.githubusercontent.com/65340361/178800384-f8c10559-25c8-4dc1-8aab-255c693e2918.png)

![Screenshot_20220713-233012](https://user-images.githubusercontent.com/65340361/178800404-7b3d2b3c-2016-4eb8-a831-d58619c8f571.png)

![Screenshot_20220713-232532](https://user-images.githubusercontent.com/65340361/178800412-98c7eda4-77ec-401a-94c1-cddbdf24d95c.png)

![Screenshot_20220713-232542](https://user-images.githubusercontent.com/65340361/178800417-ae5bdfbb-c4fe-4c59-bf30-71e3b017e7b0.png)

**Comment by HikaruIchijyo** [comment]
> Playlist implementation is under active development

Online implementations will come? Like fetch songs from yt etc?

**Comment by posLop** [comment]
> > Playlist implementation is under active development
> 
> Online implementations will come? Like fetch songs from yt etc?

I dont think thats the goal of this app.  if you need to download stuff from online playlists try spotiflyer.  it can download from yt music and spotufy playlists.  Howl is more for playing songs you already have downloaded

**Comment by Iamlooker** [comment]
> I dont think thats the goal of this app

Exactly, but I am planning to add module support, i.e. you can install modular apps which allows you to add features to howl like online playback. Something like termux

**Comment by Iamlooker** [comment]
> Exactly, but I am planning to add module support, i.e. you can install modular apps which allows you to add features to howl like online playback. Something like termux

By Modular apps I mean extensions like Tachiyomi have.

They will be able to fetch, create, edit songs in any possible way which they like and Howl will simple fetch the list provided in the end. 😉

**Comment by posLop** [comment]
> > Exactly, but I am planning to add module support, i.e. you can install modular apps which allows you to add features to howl like online playback. Something like termux
> 
> By Modular apps I mean extensions like Tachiyomi have.
> 
> They will be able to fetch, create, edit songs in any possible way which they like and Howl will simple fetch the list provided in the end. wink

that would be a super cool feature.  i look forward to future updates
